src/FileSystem/FilesFinder.php

```diff
@@ -48,7 +51,14 @@ public function findInDirectoriesAndFiles(array $source, array $suffixes = [], b
 
         $filteredFilePaths = array_filter(
             $filteredFilePaths,
-            fn (string $file): bool => ! $this->isStartWithShortPHPTag(FileSystem::read($file))
+            function (string $file): bool {
+                if ($this->isStartWithShortPHPTag(FileSystem::read($file))) {
```

Comment:
We should have a test whether this works for files using a byte order mark (BOM)

Reply:
That could be done in separate go. Care for PR? :)